src/Components/OATPropertyEditor/PropertyList.tsx

```diff
@@ -204,12 +200,16 @@ export const PropertyList = ({
         setLastPropertyFocused(null);
         const newModel = deepCopy(model);
         newModel[propertiesKeyName].splice(index, 1);
+        const dispatchDelete = () => {
+            dispatch({
+                type: SET_OAT_PROPERTY_EDITOR_MODEL,
+                payload: newModel
```

Comment:
Can you help me understand what's going on here? Why would setting the model delete it?

Reply:
The newModel has the property deleted before updating the object again, this is the way Milton implemented the original logic I'm just adding the confirmation dialog before deletion

Reply:
Ah! I see. I thought we were deleting the model. We're just deleting a property from the model.